packages/orchestrator-ui-components/src/pages/workflow/WFOStartWorkflowPage.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import { JSONSchema6 } from 'json-schema';
+import { useRouter } from 'next/router';
+import {
+    EuiFlexGroup,
+    EuiFlexItem,
+    EuiPanel,
+    EuiText,
+    EuiHorizontalRule,
+} from '@elastic/eui';
+
+import { TimelineItem, WFOLoading } from '../../components';
+import { WFOProcessDetail } from '../processes/WFOProcessDetail';
+import { ProcessDetail, ProcessStatus, StepStatus } from '../../types';
+import UserInputFormWizard from '../../components/WFOForms/UserInputFormWizard';
+import { FormNotCompleteResponse } from '../../types/forms';
+import {
+    EngineStatus,
+    useAxiosApiClient,
+    useOrchestratorTheme,
+} from '../../hooks';
+import { PATH_PROCESSES } from '../../components';
+import { getStyles } from '../../components/WFOSteps/getStyles';
+import { WFOStepStatusIcon } from '../../components/WFOSteps/WFOStepStatusIcon';
+import { useTranslations } from 'next-intl';
+
+type StartCreateWorkflowPayload = {
+    product: string;
+};
+type StartModifyWorkflowPayload = {
+    subscription_id: string;
+};
+
+type StartWorkFlowPayload =
+    | StartCreateWorkflowPayload
+    | StartModifyWorkflowPayload;
+
+interface WFOStartWorkflowPageProps {
+    workflowName: string;
+    startWorkflowPayload: StartWorkFlowPayload;
+}
+
+export interface UserInputForm {
+    stepUserInput?: JSONSchema6;
+    hasNext?: boolean;
+}
+
+export const WFOStartWorkflowPage = ({
```

Comment:
This page looks a bit big for a page, try to move some logic in separate components. Wasn't the original setup with the "CreateForm" component not better in that way? Also suggesting to call it WFOForm instead of CreateForm